environment.yml

```diff
@@ -10,10 +10,12 @@ dependencies:
   - dask
   - pip:
     - pytest
+    - imagecodecs
     - intake # for reading scivision
     - torch==1.10.0 # install before cefas_scivision; it needs this version
     - scivision
     - scikit-image
     - setuptools==69.5.1 # because this bug https://github.com/pytorch/serve/issues/3176
+    - tiffile
     - git+https://github.com/alan-turing-institute/plankton-cefas-scivision@main # torch version
     - chromadb
```

Comment:
The latest version of `chromadb` (0.5.4) causes `test_vector_store` to fail with an `AttributeError` because for whatever reason `chromadb.api.models.Collection` no longer has an attribute `model_fields`. I'm none the wiser after reading the [related issue](https://github.com/langchain-ai/langchain/issues/24163) and don't really want to dig deeper so for now we can just move forward by pinning `chromadb==0.5.3`.

Reply:
This looks rooted a transitory issue in `chromadb` related to version back-incompatibility in `pydantic` (data validation through type checking) - reading https://github.com/chroma-core/chroma/issues/2503 that team is firmly on the case and 0.5.5 should cause this problem to solve itself (always having version pinned dependencies would be a healthy habit i should adopt, though)

Reply:
I don't think there's much that can be done when this sort of thing happens, unless you're not really interested in broad support and can just lock down all your dependencies, in which case it makes sense to use lockfiles (#14) ! Good to know they're fixing it though.